### ISSUE TITLE
# in the fasta header confused the EDTA run

### ISSUE THREAD

**Issue by baozg** [oushujun/EDTA#315]
Hi, Shujun

If people follow the [PanSN-spec](https://github.com/pangenome/PanSN-spec) to set `#` as the delimiter for pangenome of different haplotype-resolved assemblies, the EDTA pipeline cannot run it properly. For now, the easiest way is to keep the header as simple as possible. But maybe the `fasta.mod` can add some code to check for strange characters in the fasta header, not only the length limits?

**Comment by oushujun** [comment]
Good call! I added a line to replace special characters with `_`, if there are multiple consecutive special characters are found, only one `_` is used to replace them.
